onboarding/src/Components/EmployeeProfile/EmployeeProfileUser.js

```diff
@@ -2,66 +2,87 @@ import React from "react";
 import { Link } from "react-router-dom";
 
 
-function EmployeeProfileUser(props) {
+function EmployeeProfileUser({ user }) {
     let avatar = "/onboarding/static/images/unknown-profile.jpg";
-    if(props.user.avatar && props.user.avatar.length > 1)
-        avatar = props.user.avatar;
+    if(user.avatar && user.avatar.length > 1)
+        avatar = user.avatar;
+    // let avatarComponent, nameLink, emailLink;
 
-    let avatarComponent, nameLink, emailLink;
+    // if(typeof props.loggedUser !== 'undefined'){
+    //     avatarComponent = <img src={ avatar } alt="avatar" /> ;
+        
+    //     nameLink = props.user.name;
 
-    if(typeof props.loggedUser !== 'undefined'){
-        avatarComponent = <Link to={{ pathname: "/employee_profile", state: { packageId: props.packageId, loggedUser: props.loggedUser, user: props.user } }} className="user-avatar user-avatar-xl">
-                             <img src={ avatar } alt="avatar" /> <span className="avatar-badge idle" title="idle"></span>
-                         </Link>;
+    //     emailLink = props.user.email;
+        // avatarComponent = <Link to={{ pathname: `/employee/${props.user.id}`, state: { packageId: props.packageId, user: props.user } }} className="user-avatar user-avatar-xxl">
+        //                      <img src={ avatar } alt="avatar" /> <span className="avatar-badge idle" title="idle"></span>
```

Comment:
No more functionality that clicking somewhere on profile box
goes back to main profile page.
Is it worth it?

Reply:
To go back to main profile page serves button "Wstecz". If it is important to have more places for this functionality, I can modify code.